google_compute_engine_oslogin/bin/google_oslogin_control

```diff
@@ -65,24 +88,29 @@ overwrite_file() {
 
 remove_from_config() {
   config=$1
-  sed -i "/${added_comment}/,+1d" ${config}.new
+  $sed_bin -i "/${added_comment}/,+1d" ${config}.new
 }
 
 remove_from_nss_config() {
-  sed -i '/^passwd:/ s/ cache_oslogin oslogin//' ${nss_config}.new
-  sed -i '/^passwd:/ s/ cache oslogin//' ${nss_config}.new
-  sed -i '/^passwd:/ s/ oslogin//' ${nss_config}.new
+  $sed_bin -i '/^passwd:/ s/ cache_oslogin oslogin//' ${nss_config}.new
+  $sed_bin -i '/^passwd:/ s/ cache oslogin//' ${nss_config}.new
+  $sed_bin -i '/^passwd:/ s/ oslogin//' ${nss_config}.new
 }
 
 add_to_sshd_config() {
   remove_from_config ${sshd_config}
-  sed -i "\$a${added_comment}\n${sshd_command}" ${sshd_config}.new
-  sed -i "\$a${added_comment}\n${sshd_user}" ${sshd_config}.new
+  $sed_bin -i "\$a${added_comment}\n${sshd_command}" ${sshd_config}.new
+  $sed_bin -i "\$a${added_comment}\n${sshd_user}" ${sshd_config}.new
 }
 
 add_to_nss_config() {
   remove_from_nss_config
-  sed -i '/^passwd:/ s/$/ cache_oslogin oslogin/' ${nss_config}.new
+  $sed_bin -i '/^passwd:/ s/$/ cache_oslogin oslogin/' ${nss_config}.new
+
+  if is_freebsd; then
+    # Replace compat by files (as compat cannot be used with other sources)
```

Comment:
Please start comments with capital letters and end comments with periods.

Reply:
Fixed.